examples/nlp/language_modeling/bert_pretraining.py

```diff
@@ -423,8 +425,8 @@ def create_pipeline(data_file, batch_size, preprocessed_data=False, batches_per_
     step_freq=args.save_step_freq,
 )
 
-ckpt_eval = nemo.core.EvaluatorCallback(
-    eval_tensors=[eval_mlm_loss, eval_nsp_loss],
+eval_callback = nemo.core.EvaluatorCallback(
+    eval_tensors=eval_log_tensors,
```

Comment:
I could not comment on the next section. Why not calculate total_steps outside of the if condition to have just one  lr_policy_fn = get_lr_policy()?